_data/users.yml

```diff
@@ -734,6 +734,29 @@
     - urltext: Installing Trino with Plural
       url: https://docs.plural.sh/applications/trino
   # supplied by developer advocate https://github.com/avaidyanatha
+- name: Quora
+  highlight: false
+  anchor: quora
+  logo: /assets/images/logos/quora.png
```

Comment:
Can you add those images in 500x400 and 250x200 resolution

Reply:
Done, updated them